cmd/node/node.go

```diff
@@ -436,6 +434,15 @@ func (app *SpacemeshApp) startServices() {
 		log.Panic("cannot start block producer")
 	}
 	app.poetListener.Start()
+
+	if app.Config.StartMining {
```

Comment:
Line 400:
```
if app.Config.CoinbaseAccount == "" {
  app.log.Panic("cannot start mining without Coinbase account")
}
```

If `StartMining` is false then you should not panic.

Reply:
This wasn't fixed yet.
And after what you did here:
```
if coinBase.Big().Uint64() == 0 && app.Config.StartMining {
   app.log.Panic("invalid Coinbase account")
}
```
It should probably get removed entirely.